.travis/yapf.sh

```diff
@@ -51,6 +51,13 @@ format_changed() {
         git diff --name-only --diff-filter=ACM "$MERGEBASE" -- '*.py' | xargs -P 5 \
              yapf --in-place "${YAPF_EXCLUDES[@]}" "${YAPF_FLAGS[@]}"
     fi
+
+    if which clang-format >/dev/null; then
+        if ! git diff --diff-filter=ACM --quiet --exit-code "$MERGEBASE" -- '*.cc' '*.h' &>/dev/null; then
+            git diff --name-only --diff-filter=ACM "$MERGEBASE" -- '*.cc' '*.h' | xargs -P 5 \
+                 clang-format -i
+        fi
+    fi
```

Comment:
This probably doesn't belong in a script called `yapf.sh`. It's a bit redundant in the ci since we run both `.travis/check-git-clang-format-output.sh` and `.travis/yapf.sh --all`, but I'm guessing you're adding this here to use locally?